marklogic-data-hub/src/main/java/com/marklogic/hub/flow/RunFlowResponse.java

```diff
@@ -25,8 +25,9 @@
     public List<String> completedItems;
     public List<String> failedItems;
     public List<JsonNode> errors;
+    public List<String> documents;
 
     public String toString() {
-        return "{totalCount: " + totalCount + ", errorCount: " + errorCount + ", completedItems: " + completedItems.size() + ", failedItems: " + failedItems.size() + ", errors: " + errors.size() + "}";
+        return "{totalCount: " + totalCount + ", errorCount: " + errorCount + ", completedItems: " + completedItems.size() + ", failedItems: " + failedItems.size() + ", errors: " + errors.size() + ", documents: " + documents.size() + "}";
     }
```

Comment:
Could it be better using 
String.format("{totalCount: %d, errorCount: %d, completedItems: %d, failedItems: %d, errors: %d, documents: %d}",  totalCount, errorCount, completedItems.size(), failedItems.size(), errors.size(), documents.size());